app.js

```diff
@@ -39,6 +40,18 @@ var vcapServices = require( 'vcap_services' );
 
 var bankingServices = require('./banking_services');
 
+const DEFAULT_NAME = "watson-banking-chatbot";
+const DISCOVERY_ACTION = "rnr";  // Replaced RnR w/ Discovery but Conversation action is still "rnr".
+// Discovery environment and collection are found by name if ID is not provided.
+const DISCOVERY_ENVIRONMENT_NAME = process.env.DISCOVERY_ENVIRONMENT_NAME || DEFAULT_NAME;
+const DISCOVERY_COLLECTION_NAME = process.env.DISCOVERY_COLLECTION_NAME || DEFAULT_NAME;
+const DISCOVERY_DOCS = [

```

Comment:
also, i'm still very confused by the different things in the data folder, let's delete things we don't need.